Rakefile

```diff
@@ -101,7 +101,8 @@ def package
     system(
       "zip", "-q", "-r", "--symlinks",
       "tuist.zip", "tuist",
-      "ProjectDescription.swiftmodule", "ProjectDescription.swiftdoc", "libProjectDescription.dylib", "ProjectDescription.swiftinterface"
+      "ProjectDescription.swiftmodule", "ProjectDescription.swiftdoc", "libProjectDescription.dylib", "ProjectDescription.swiftinterface",
+      "../../Templates"
```

Comment:
I'd avoid paths relative to the path `rake` is executed from. You can replace `../../Templates` with `File.expand_path("Templates", __dir__)`